### PR TITLE
391 emergency button

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/W6IaMMTA/391-vv-e-design-emergency-information-button-and-popup

## Description

- Most of the design done, and some temporary copy

@neontribe/sea-map

### CODE REVIEW

File: src/Theme.elm [hunk 1]
```diff
@@ -265,4 +269,130 @@ containerContent children =
 
 page : List (Html msg) -> Html msg
 page children =
```
Comment (katjam) [inline src/Theme.elm:271]: Will maybe move this out in a next task.

File: src/Theme.elm [hunk 2]
```diff
@@ -265,4 +269,130 @@ containerContent children =
...
+        ([ globalStyles
+         , button [ css [ emergencyButtonStyle ] ]
+            [ span [] [ text (t EmergencyButton) ]
+            , img [ css [ iconStyle ], src "Emergency.svg", alt "" ] []
+            ]
+         ]
+            ++ children
+            ++ [ div [ css [ emergencyPanelStyle ], id "emergency" ]
+                    [ div [ css [ emergencyPanelHeaderStyle ] ]
+                        [ img [ css [ emergencyPanelHeaderIconStyle ], src "Emergency.svg", alt "" ] []
```
Comment (katjam) [inline src/Theme.elm:282]: Text to move to copy file in implementation of function stage.

File: src/Theme.elm [hunk 3]
```diff
@@ -265,4 +269,130 @@ containerContent children =
...
+                        , div [ css [ emergencyPanelHeaderTextStyle ] ] [ text "Are you in danger right now?" ]
+                        , button [ css [ emergencyPanelHeaderButtonStyle ] ] [ img [ css [ emergencyPanelHeaderIconStyle ], src "Arrow.svg", alt "" ] [] ]
+                        ]
+                    , div [ css [ emergencyPanelBodyStyle ] ]
+                        [ p [] [ text "If you need emergency help call:" ]
+                        , ul [ css [ listStyle none, margin2 (rem 1) zero ] ]
+                            [ li [ css [ marginBottom (rem 1) ] ] [ text "Police ", a [ href "#" ] [ text "999" ] ]
```
Comment (katjam) [inline src/Theme.elm:289]: Do we want these defined inline or as style functions like the others?

File: src/Theme.elm [hunk 4]
```diff
@@ -265,4 +269,130 @@ containerContent children =
...
+        , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+            [ borderRadius zero
+            ]
+        ]
+
+
+emergencyButtonStyle : Style
+emergencyButtonStyle =
+    batch
+        [ backgroundColor white
```
Comment (katjam) [inline src/Theme.elm:372]: alphabetics?

**Review by katjam — COMMENTED** [review]
Ready to approve after icon.

File: src/Theme.elm (hunk 1 above)
Comment (Nikomus) [reply src/Theme.elm:271]: Yep, I think the function, plus exit button markup/style/function should all be with this and in a pagetemplate file

**Review by katjam — APPROVED** [review]
If you are happy with the inline styles I'm happy for this to go in. :snowboarder:

**Pull request merged**